### PR TITLE
Default auto field

### PR DESCRIPTION
**Pull request by markrawls**

### CODE REVIEW

File: tests/middleware/conftest.py [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 
 def pytest_configure():
-    settings.configure(SESSION_ENGINE="django.contrib.sessions.backends.file")
+    settings.configure(
+        SESSION_ENGINE="django.contrib.sessions.backends.file", SECRET_KEY="required"
```
Comment (cviveiro) [inline tests/middleware/conftest.py:6]: naive q: why this change?

**Review by cviveiro — COMMENTED** [review]
THANK YOU FOR THIS

File: tests/middleware/conftest.py (hunk 1 above)
Comment (markrawls) [reply tests/middleware/conftest.py:6]: Because instead of generating a secret key when none is provided in test suites it just throws an error now saying that it needs one. We don't wanna actually ship one with the library cuz that would be a security risk for people that never change the defaults, so we put one in the test suite instead.

**Review by cviveiro — APPROVED** [review]
